frontend/src/components/form/index.tsx

```diff
@@ -29,15 +33,16 @@ Form.Title = function FormTitle({ children, ...restProps }: any) {
   return <Title {...restProps}>{children}</Title>
 }
 
-Form.Input = function FormInput({ setInput, ...restProps }: any) {
+Form.Input = function FormInput({ handleInputChange, ...restProps }: any) {
   return (
     <Input
       marginY={[1, 2]}
       padding={[1, 2]}
       {...restProps}
-      onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
-        setInput(event.target.value)
-      }
+      onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
If there will be no additional logic here, you could simply write: 

`onChange={onChange}`. And you can go one step further (if you name the prop `onChange`) and let it spread with `restProps`